swagger.yaml

```diff
@@ -4782,7 +4782,7 @@ definitions:
         enum: ['Full', 'None']
         default: 'Full'
         description: |
```

Comment:
```suggestion
        default: 'Full'
        deprecated: true
        description: |
```

Reply:
Marking definitions is API 2.0 is not supported yet - https://stackoverflow.com/questions/49870109/how-to-annotate-a-field-as-deprecated-in-openapi-swagger-2-0 (also got an error trying myself)